components/WarningNote/WarningNotes.spec.tsx

```diff
@@ -13,33 +28,44 @@ jest.mock('next/router', () => ({
 }));
 
 describe(`useWarningNotes`, () => {
-  it('should render a list of warning notes', () => {
-    jest.spyOn(warningNotes, 'useWarningNotes').mockImplementation(() => ({
-      data: mockedWarningNote,
-      mutate: jest.fn(),
-      revalidate: jest.fn(),
-      isValidating: false,
-    }));
-
-    const { asFragment } = render(<WarningNotes id={123} />);
-    expect(asFragment()).toMatchSnapshot();
-  });
-
-  it('should not render the review date if there is a scheduled future review date', () => {
+  beforeEach(() => {
     jest.spyOn(warningNotes, 'useWarningNotes').mockImplementation(() => ({
       data: [
         warningNoteFactory.build({
-          nextReviewDate: '2020-11-12',
+          nextReviewDate: '2020-12-30',
```

Comment:
Why do we change the date here? e.g. preference? Date weirdness in tests? 😁

Reply:
@Renny4Real: @martapederiva had exactly the same question, and I didn't have a good answer 😅 . I'm going to take a look and will report back, it's possible I changed it in error!

Reply:
@Renny4Real @martapederiva: Looked like an oversight. Restoring it to the original date, things are still working as expected 👍